packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -145,7 +152,10 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
       const componentTypes = node?.components.map((component) => component.type) ?? [];
 
       const tagComponent = findComponentByType(node, KnownComponentType.Tag) as IAnchorComponentInternal;
-
+      const entityBindingComponent = findComponentByType(
```

Comment:
`componentTypes` and `additionalComponentData` is supposed to match, meaning if the first element in `componentTypes` is `EntityBinding`, then the first element in `additionalComponentData` should also be the additional data for entity binding. so you should update the logic below to enforce that instead of always adding entity binding data later

Reply:
I did not follow your comment. Could you give me an example?